tools/clang_json/update_compile_database.cc

```diff
@@ -0,0 +1,55 @@
+#include <fstream>
+#include <string>
+
+#include <unistd.h>
+
+#include "absl/strings/str_cat.h"
+#include "absl/strings/str_join.h"
+
+#include "google/protobuf/io/coded_stream.h"
+#include "google/protobuf/io/zero_copy_stream_impl.h"
+#include "google/protobuf/stubs/common.h"
+
+#include "src/main/protobuf/extra_actions_base.pb.h"
+
+#include "include/json/json.h"
+
+int main(int argc, char **argv) {
```

Comment:
Can you add some documentation for what this does?

Reply:
Done.